lib/features/wallet_screen/wallet_screen.dart

```diff
@@ -0,0 +1,80 @@
+part of '../feature.dart';
+
+class WalletScreen extends ConsumerWidget {
+  const WalletScreen({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final isMobile = context.isMobile; 
+    final isPortrait = context.isPortrait; 
+    return Scaffold(
+      appBar: isMobile ? MobileHeader(title: 'WALLET') : null,
+      backgroundColor: context.primaryBackgroundColor,
+      floatingActionButton:
+          isMobile || isPortrait ? _floatingActionButton(context) : SizedBox(),
+      floatingActionButtonLocation: FloatingActionButtonLocation.centerFloat,
+      body: SafeArea(
+        child: LayoutBuilder(
+          builder: (context, constraints) {
+            final double maxWidth = AppValues.width600;
+            final double maxWidthTablet = AppValues.width1440;
+            return isMobile
+                ? ConstrainedBox(
+                    constraints: BoxConstraints(maxWidth: maxWidth),
+                    child: Padding(
+                      padding: const EdgeInsets.symmetric(
+                          horizontal: AppValues.padding16),
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          verticalSpace(30),
+                          WalletScreenBody(),
+                          Spacer(),
+                        ],
+                      ),
+                    ),
+                  )
+                : ConstrainedBox(
+                    constraints: BoxConstraints(maxWidth: maxWidthTablet),
+                    child: Padding(
+                      padding: const EdgeInsets.only(left: 32, right: 80),
+                      child: WalletScreenTabletMode(),
+                    ),
+                  );
+          },
+        ),
+      ),
+    );
+  }
+
+  Widget _floatingActionButton(BuildContext context) {
```

Comment:
also this FAB should be repeated in multiple screen. add it to component (`component/fab`) and replace all existing `new wager` fab with the component